packages/ui/components/scheduler/components/currentTime/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from "react";
+
+import dayjs from "@calcom/dayjs";
+
+import { useSchedulerStore } from "../../state/store";
+
+type Props = {
+  containerNavRef: React.RefObject<HTMLDivElement>;
+  containerRef: React.RefObject<HTMLDivElement>;
+  containerOffsetRef: React.RefObject<HTMLDivElement>;
+};
+
+export function CurrentTime({ containerOffsetRef }: Props) {
+  const [currentTimePos, setCurrentTimePos] = useState<number>(0);
+  const { startHour, endHour } = useSchedulerStore((state) => ({
+    startHour: state.startHour || 0,
+    endHour: state.endHour || 23,
+  }));
+
+  useEffect(() => {
+    // Set the container scroll position based on the current time.
+    let currentMinute = new Date().getHours() * 60;
+    currentMinute = currentMinute + new Date().getMinutes();
```

Comment:
Need to figure out what the difference in minutes is between the current time and the time that is available in the grid